### ISSUE TITLE
Integration broke in 2021.12

### ISSUE THREAD

**Issue by grinco** [Limych/ha-jq300#74]
Setup failed for jq300: Unable to import component: cannot import name 'HTTP_OK' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)

**Comment by elik745i** [comment]
what to do?

**Comment by Kurtroosbeek2** [comment]
Solution:
Edit the file:
/config/custom_components/jq300/api.py

Around line 20 add the from line:

```
import async_timeout
from http import HTTPStatus
import homeassistant.util.dt as dt_util
```

Around line 25 change the code to delete the HTTP status OK.
```
from homeassistant.const import (
    CONCENTRATION_MILLIGRAMS_PER_CUBIC_METER,
    CONCENTRATION_PARTS_PER_BILLION,
    CONCENTRATION_PARTS_PER_MILLION,
    CONF_UNIT_OF_MEASUREMENT,
)
```


Around Line 216 change the code to this:
```
        if (
            response.status != HTTPStatus.OK and response.status != HTTP_NO_CONTENT
        ):  # pragma: no cover
            _LOGGER.debug(MSG_GENERIC_FAIL)
            return None
```

 That solved it for me :)

**Comment by Limych** [comment]
Fixed in #73
